### PR TITLE
Contribution Guidelines explained better

### PR DESCRIPTION
**Pull request by sayantani11**
The part explaining how we are supposed to add
the commit message by git can be a bit confusing
for some new contributors as most people tend to
do it with github and not git. 
Added few lines in theCONTRIBUTING.md file,
which explains the procedure a little better
to avoid any confusion. Also explains
procedure if the contributor wants to make
any changes to their existing git commit message.

Resolves issue #1019

Signed-off-by: Sayantani Saha <ii.sayantani.ii@gmail.com>

### CODE REVIEW

**Comment by sayantani11** [comment]
@rnjudge I tried to keep it as simple as possible, if any further changes are required please let me know.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -219,6 +219,15 @@ The commit message of your PR should be able to communicate what problem/opportu
 - Wrap the body at 72 characters
 - Use the body to explain what and why vs. how
 
+The commit message should be done with git and for that run the following command in your terminal:
+
+```sh
+$ git commit
```
Comment (rnjudge) [inline CONTRIBUTING.md:225]: Can you change to `git commit <files changed>` ? I don't think just `git commit` will work.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -268,6 +277,15 @@ See also: #456, #789
 Signed-off-by: Some Dev <somedev@example.com>
 ```
 
+To make any changes to your existing git commit message, run the following command in your terminal:
```
Comment (rnjudge) [inline CONTRIBUTING.md:280]: Change the section after the comma to: `make sure you are on your development branch and run the following command in your terminal`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -219,6 +219,15 @@ The commit message of your PR should be able to communicate what problem/opportu
 - Wrap the body at 72 characters
 - Use the body to explain what and why vs. how
 
+The commit message should be done with git and for that run the following command in your terminal:
+
+```sh
+$ git commit
+```
+
+Once this command is executed, you will be able to add the commit message(follow the guidelines given below for this).
```
Comment (rnjudge) [inline CONTRIBUTING.md:228]: small nit: add a space before `(`

File: CONTRIBUTING.md (hunk 1 above)
Comment (sayantani11) [reply CONTRIBUTING.md:225]: I guess that's for git add
Even for this changes I did the following:
```sh
$ git add <file name>
```

```sh
$ git commit
```

File: CONTRIBUTING.md (hunk 2 above)
Comment (sayantani11) [reply CONTRIBUTING.md:280]: On it

File: CONTRIBUTING.md (hunk 1 above)
Comment (rnjudge) [reply CONTRIBUTING.md:225]: Doh! @sayantani11 you are right :) Apologies, I was definitely thinking about git add when I made that comment. This looks good.

**Review by rnjudge — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (sayantani11) [reply CONTRIBUTING.md:225]: Hahaha..... Happens @rnjudge

**Pull request merged**